internal/server/auth/common/common.go

```diff
@@ -0,0 +1,9 @@
+package common
+
+// RequestDetails is a type representing an authorization request.
+type RequestDetails struct {
+	UserName             string
```

Comment:
Tiny nit, but lets make that `Username` to line up with our usual spelling across the codebase (there are two exceptions currently, but neither are user visible).